crates/re_viewport/src/viewport.rs

```diff
@@ -238,6 +238,11 @@ impl<'a, 'b> Viewport<'a, 'b> {
         let executed_systems_per_space_view =
             execute_systems_for_space_views(ctx, tree, &blueprint.space_views);
 
+        let mut contents_per_tile_id = HashMap::default();
+        blueprint.for_each_contents(&mut |contents| {
+            contents_per_tile_id.insert(contents.as_tile_id(), *contents);
+        });
```

Comment:
Is there a reason we need to do this as a walk?

The ViewportBlueprint already has flat lists of both containers and space-views that we could iterate over directly.

Reply:
Oh right, I didn't think of that 🤦🏻